version/version.go

```diff
@@ -0,0 +1,7 @@
+package version
+
+// These are set through ldflags='-X ...' in the Makefile
+var (
+	TaggedRelease string
+	CommitHash    string
+)
```

Comment:
Starting from Go 1.18, I believe the go toolchain automatically embeds VCS information into the binary. Grep for buildvcs and buildinfo here: https://tip.golang.org/doc/go1.18

Maybe we could use that instead? Does that provide us with enough info? For example, I'm not sure if it captures tags.

Reply:
Unfortunately `BuildInfo.Settings` doesn't include git tags. That shell command in the makefile was the simplest though, so at least we get rid of the complicated command to get the commit hash.